src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -310,12 +310,12 @@ export function QueryPageContainerComponent(): JSX.Element {
             }
         }
 
-        if (chooseExportContent.raw) {
+        if (exportOptions.raw) {
             const urlParams = new URLSearchParams(history.location.search);
             urlParams.delete("row");
             urlParams.delete("column");
             const urlParamsString = urlParams.toString();
-            const isolateFilteredUrl = `${ISOLATE_URL}/${urlParamsString}`;
+            const isolateFilteredUrl = `${ISOLATE_URL}?${urlParamsString}`;

```

Comment:
Was this a change in the api? Or a bug?

Reply:
Change in the api that I didn't know about until now.

Reply:
I think he was not aware of those changes. You could tell him to test the client after pushing changes to the server code, so that he at least can tell you that some changes in the client are necessary.